lib/main_containers.pm

```diff
@@ -84,23 +92,44 @@ sub load_host_tests_docker {
     loadtest "containers/container_diff" if (is_opensuse());
 }
 
+sub load_jeos_containers {
+    loadtest 'containers/podman';
+    loadtest 'containers/podman_image';
+    loadtest 'containers/podman_3rd_party_images';
+    loadtest 'containers/docker';
+    loadtest 'containers/docker_runc';
+    loadtest 'containers/docker_3rd_party_images';
+    loadtest 'containers/docker_image';
+    loadtest 'containers/docker_compose';
+    loadtest 'containers/registry' if is_x86_64;
+    loadtest 'containers/zypper_docker';
+    loadtest 'containers/rootless_podman';
+}
 
 sub load_container_tests {
     my $runtime = get_required_var('CONTAINER_RUNTIME');
     if (get_var('BOOT_HDD_IMAGE')) {
         loadtest 'installation/bootloader_zkvm' if is_s390x;
-        loadtest 'boot/boot_to_desktop';
+        loadtest 'boot/boot_to_desktop' if (is_container_jeos || !is_jeos);
```

Comment:
By default we always run `boot/boot_to_desktop`, so this logic is flawed to fail in the future.
The default should always be performed `unless` the exception and not the other way around. So instead of asking "right now, boot_to_desktop needs to be executed if ...." one should approach it as "boot_to_desktop should always be executed, except for the special case, that ....

TL'DR: Please change it to something like
```suggestion
        loadtest 'boot/boot_to_desktop' unless ... ;
```

Reply:
but remains the default.  i tried with unless but there were tests failing. i would like to leave it as it is, because this is getting complex enough and i think it is going to be a better approach soon after.

Reply:
It does not remain the default, because it is only executed if a certain condition is fulfilled.
The opposite is that it is executed by default, unless a condition is fulfilled.

The difference is opt-in by default or opt-out by default.

Why is this important? Because if we add another test e.g. for SLE Micro, that one would not execute `boot/boot_to_desktop` unless we add that routine to the `if` loop.
If we have a exclusion logic (`unless`) then every new test runs, however weird they might be, will use this by default unless specified otherwise.